### ISSUE TITLE
Randomization Error

### ISSUE THREAD

**Issue by jonwelsch** [FIRST-Tech-Challenge/scorekeeper#454]
Setup from this weekend:

2019 MacBook Pro (15") running Catalina 10.15.2 running the server
Surface Pro 6 on current windows 10 running match control.
2 Fields

Field 1 never randomized different, field 2 always randomized different. An exit and reload of the match control page fixed things. 

Two additional different errors on this setup, will link the tickets.
#456 #455

**Comment by adwiii** [comment]
Do you have the db/name for the event?

**Comment by jonwelsch** [comment]
North Georgia Meet #3

**Comment by adwiii** [comment]
I was able to pull the event data from the upload. While it is rare that the event began with rolling five 5s in a row, the overall randomization statistics of the event don't seem to indicate a problem with the fairness/randomness of the rolls. The randomization for the match is performed in the server itself and not on the control page, so it is likely coincidence that closing and re-opening the control page preceded the randomization coming up different. I ran a [Pearson's Chi-Squared Test](https://en.wikipedia.org/wiki/Pearson's_chi-squared_test) on the randomization, and it is within a normal range. (Chi-Squared: 6.714286, Probability of having Chi-Squared higher than that for a fair 6 sided die: .24)

We have some more statistics in issue #320 if you are interested. We have tested the randomness generation and are reasonably confident that it is balanced.
